README.md

```diff
@@ -1,45 +1,76 @@
 # Sentry Release GitHub Action
-Automatically create a Sentry Release in a workflow.
-
-## Why Releases?
-A release is a version of your code that is deployed to an environment. 
-When you give Sentry information about your releases, you unlock a number of new features:
- - Determine the issues and regressions introduced in a new release
- - Predict which commit caused an issue and who is likely responsible
- - Resolve issues by including the issue number in your commit message
- - Receive email notifications when your code gets deployed
-
-Additionally, releases are used for applying source maps to minified JavaScript to view original, untransformed source code.
+Automatically create a Sentry release in a workflow. 
+You can learn more about releases in [releases documentation](https://docs.sentry.io/workflow/releases).
 
 ## Usage
 Adding the following to your workflow will create a new Sentry release for the 
-`production` environment of your project and upload JavaScript source maps from
-the `./lib` directory. 
+`production` environment of your project with an automatically generated `version` name.
   
 ```yaml
-uses: getsentry/action-release@v1
-with:
-  environment: 'production'
-  sourcemaps: './lib'
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+      SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+      SENTRY_PROJECT: ${{ secrets.SENTRY_PROJECT }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: getsentry/action-release@v1.0.0
+      with:
+        environment: production
 ```
 
-Don’t forget to include the following environment variables:
-```yaml
-env:
-  SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
-  SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
-  SENTRY_PROJECT: ${{ secrets.SENTRY_PROJECT }}
-```
+### Inputs
+#### Parameters
+|Name|Description|
+|---|---|
+|`environment`|_Required_. Set the environment for this release. E.g. "production" or "staging".|
+|`finalize`|When false, omit marking the release as finalized and released.|
+|`sourcemaps`|Space-separated list of paths to JavaScript sourcemaps. Omit to skip uploading sourcemaps.|
+|`started_at`|Unix timestamp of the release start date. Omit for current time.|
+|`version`|Identifier that uniquely identifies the releases. Omit to auto-generate one.|
 
-See [action.yml](https://github.com/getsentry/action-release/blob/master/action.yml) for more workflow options.
+#### Environment Variables
+The following are all _required_.
 
-## Troubleshooting
-Building and running this action locally on an unsupported environment yields an error that looks like:
-```
-Syntax error: end of file unexpected (expecting ")")
-```
+|Name|Description|
+|---|---|
+|`SENTRY_AUTH_TOKEN`|See [authentication documentation](https://docs.sentry.io/api/auth). |
```

Comment:
I don't think we want to be pointing people down this path. This points people to generate auth tokens associated with users. We should point people to create internal integrations.

Example: https://github.com/getsentry/sentry-netlify-build-plugin#create-a-sentry-internal-integration

Reply:
This is a really good point. For now I'll copy in what we have for Netlify.